sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1668,9 +1668,11 @@ def _flush_batch(self, destination):
     _LOGGER.debug(
         'Flushing data to %s. Total %s rows.',
         destination,
-        len(rows_and_insert_ids))
-    self.batch_size_metric.update(len(rows_and_insert_ids))
+        len(rows_and_insert_ids_with_windows))
+    self.batch_size_metric.update(len(rows_and_insert_ids_with_windows))
 
+    rows_and_insert_ids = [r[0] for r in rows_and_insert_ids_with_windows]
```

Comment:
You can use the "zip to transpose" trick here. 

```
rows_and_insert_ids, window_values = zip(*rows_and_insert_ids_with_windows)
```

Reply:
Good call, done